### DISCUSSION TITLE
🇺🇦 FSNotes development updates 🇺🇦

### DISCUSSION THREAD

**Discussion by gingerbeardman** [glushchenko/fsnotes#1313] [General]
FSNotes project owner, Oleksandr Hlushchenko, lives in Ukraine and is currently fleeing during the Russian invasion.

> My city under fire from the Russian army. FSNotes development has been suspended indefinitely. We are in a bomb shelter. I hope they don't kill me and the release will still take place someday.

Ref: https://twitter.com/FSNotesApp/status/1496865592242016259

**Comment by gingerbeardman** [comment]
> Guys, i'm alive. Despite the fact that the Russian invaders destroyed my city, i managed to leave Kharkiv yesterday. Now I am in a safe place, still in Ukraine.

Ref: https://twitter.com/FSNotesApp/status/1499681219524575233

**Comment by gingerbeardman** [comment]
> Thanks for support guys. I managed to work on the weekend, and i will try to release an FSNotes 5 for mobile this week, until we were surrounded.

Ref: https://twitter.com/FSNotesApp/status/1500734611503996930

**Comment by glushchenko** [comment]
https://github.com/glushchenko/fsnotes/issues/1312
